flutter_local_notifications/README.md

```diff
@@ -589,6 +589,20 @@ void onDidReceiveNotificationResponse(NotificationResponse notificationResponse)
 
 In the real world, this payload could represent the id of the item you want to display the details of. Once the initialisation is complete, then you can manage the displaying of notifications. Note that this callback is only intended to work when the app is running. For scenarios where your application needs to handle when a notification launched the app refer to [here](#getting-details-on-if-the-app-was-launched-via-a-notification-created-by-this-plugin)
 
+Use the `getNotificationAppLaunchDetails` method when the app starts if you need to handle when a notification triggering the launch for an app e.g. change the home route of the app for deep-linking.
```

Comment:
Thanks for making the PR. Upon looking at what already exists, what got added here looks quite odd as there's now already a statement with a link to the section on dealing with when a notification causes an app to start (`For scenarios where your application needs to handle when a notification launched the app refer to [here](#getting-details-on-if-the-app-was-launched-via-a-notification-created-by-this-plugin)`). Besides how the readme actually mentions the dealing with this scenario requires doing something different, it reads awkwardly in that it is  immediately followed with a separate paragraph with instructions that is on the same topic. Perhaps it's better that what you added goes to the section that is linked?